templates/components/form/header_content.html.twig

```diff
@@ -100,6 +100,16 @@
             {{ nametype }}
          {% endif %}
       </span>
+      {% if item.isField('is_dynamic') and item.fields['is_dynamic'] %}
+      <span {% if in_navheader %} class="mx-1 bg-secondary status rounded-1" title="{{ __('Automatic Inventory') }}"
+                     data-bs-toggle="tooltip" {% endif %}>
+         {% if in_navheader %}
+            <i class="{{ call('Agent::getIcon') }}"></i>
+         {% endif %}
+      </span>
+      {% endif %}
+
```

Comment:
Unless I miss something, when `in_navheader === false`, proposed code will produce an empty span. I think it could be simplified.

```suggestion
      {% if in_navheader and item.isField('is_dynamic') and item.fields['is_dynamic'] %}
      <span class="mx-1 bg-secondary status rounded-1" title="{{ __('Automatic Inventory') }}" data-bs-toggle="tooltip">
         <i class="{{ call('Agent::getIcon') }}"></i>
      </span>
      {% endif %}
```